app/src/common/gecko/com/igalia/wolvic/browser/api/impl/GeckoViewFetchClient.kt

```diff
@@ -23,8 +23,8 @@ import java.util.concurrent.TimeoutException
  * GeckoView ([GeckoWebExecutor]) based implementation of [Client].
  */
 class GeckoViewFetchClient(
-    context: Context,
-    private val maxReadTimeOut: Pair<Long, TimeUnit> = Pair(MAX_READ_TIMEOUT_MINUTES, TimeUnit.MINUTES)
+        val context: Context,
```

Comment:
AFAIK all kotlin parameters are val so this would be useless

The problem here is that the parameter is unused, so add ` @Suppress("UNUSED_PARAMETER")` to the parameter.

Or even better let's just remove it.

Reply:
Thanks! I just removed it.